### PR TITLE
Script to summarize Sensorpush data

### PR DESCRIPTION
**Pull request by alneberg**
Should run daily to fetch hourly temperature averages for all sensors. Will check if any sensor is outside of its defined limits, and record the start and stop time for the outside of limit interval. For any such interval and 1 hour before and 1 hour after, temperature samples are kept in the original resolution, which seems to be 1 measurement per minute.

### CODE REVIEW

File: sensorpush_to_statusdb.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+    def __init__(self, email, password):
+        self.email = email
+        self.password = password
+        self._authorized = False
+        self.base_url = 'https://api.sensorpush.com/api/v1/'
+        self.access_token = None
+
+    def _authorize(self):
+        url_ending = 'oauth/authorize'
+        url = '/'.join(x.strip('/') for x in [self.base_url, url_ending] if x)
```
Comment (aanil) [inline sensorpush_to_statusdb.py:31]: Why not leave out the ending `/` in the base url so you don't have to strip it out later, if I'm right in assuming that's what is done here?

Comment (alneberg) [reply sensorpush_to_statusdb.py:31]: Good point, yeah I guess that would work...

Comment (alneberg) [reply sensorpush_to_statusdb.py]: Adjusted!

File: sensorpush_to_statusdb.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    if temp_alerts.get('enabled'):
+        if 'max' in temp_alerts:
+            limit_upper = to_celsius(temp_alerts['max'])
+        if 'min' in temp_alerts:
+            limit_lower = to_celsius(temp_alerts['min'])
+
+    return limit_lower, limit_upper
+
+
+def to_celsius(temp):
```
Comment (aanil) [inline sensorpush_to_statusdb.py:180]: Strange that sensorpush does not give you results in celsius

**Review by aanil — APPROVED** [review]
Had a quick look, looks mostly ok to me. I can fix things later if something doesn't work out.

**Pull request merged**